source/macos/AdaptiveCards/AdaptiveCards/Renderers/FactSetRenderer.swift

```diff
@@ -73,4 +84,24 @@ class FactSetRenderer: NSObject, BaseCardElementRendererProtocol {
         
         return mainFactView
     }
+    
+    func getMarkdownString(parserResult: ACSMarkdownParserResult) -> NSMutableAttributedString {
+        let content: NSMutableAttributedString
```

Comment:
I think TextBlock is also using this logic. We can move this in a util class and both renderer can leverage this.

Reply:
Nice one
Done!